packages/css/src/utils/__tests__/css-transform.test.tsx

```diff
@@ -132,7 +151,7 @@ describe('leading pseduos in css', () => {
     `
     );
 
-    expect(actual.join('\n')).toMatchInlineSnapshot(`"._12t41q9v >:first-child{color:hotpink}"`);
+    expect(actual.join('\n')).toMatchInlineSnapshot(`"._gkte1q9v > :first-child{color:hotpink}"`);
```

Comment:
this changed because the selector changed from

```
 >:first-child
```

to

```
 > :first-child
```

Reply:
this is another vector we need to make sure we normalize - currently the two could be different in userland but semantically the same - but be considered to be different thus generating a different group hash...

Reply:
made bug here https://github.com/atlassian-labs/compiled/issues/326